engine/lib/import_thir.ml

```diff
@@ -257,6 +257,9 @@ end) : EXPR = struct
       | Ge -> Core__cmp__PartialOrd__ge
       | Gt -> Core__cmp__PartialOrd__gt
       | Eq -> Core__cmp__PartialEq__eq
+      | Cmp ->
+          assertion_failure (Span.to_thir span)
+            "`Cmp` binary operator is not suppored"
```

Comment:
The new `Cmp` operator corresponds to [`Ord::cmp`](https://doc.rust-lang.org/std/cmp/trait.Ord.html#tymethod.cmp): it's a function that compare two values and returns `-1`, `0` or `1` depending on their ordering. It's also known as "the spaceship operator" (noted `<=>`) in C++ circles I believe.

Anyway, I expect it to show up in generated code so you'll likely have to implement it in some way.